src/app/components/LiveHeaderMedia/index.test.tsx

```diff
@@ -102,6 +102,73 @@ describe('liveMediaStream', () => {
     expect(container).toBeEmptyDOMElement();
   });
 
+  it.each([
+    {
+      title: 'Displays a title with a comma when there is no punctuation.',
+      inputTitle: 'Title with no punctuation',
+      expectedResult:
+        '<div><span class="hoverStylesText css-1hja70-mediaDescription-openMediaDescription-description"><span class="css-1cxhpmz-Text">Title with no punctuation,</span><span class="css-3g34uu-Text"> CBBC</span></span><span class="css-10lim9y-guidanceMessage" data-testid="warning-message"><span class="css-49uz87-visuallyHiddenText">, </span>Contains some upsetting scenes.</span></div><div class="hoverStylesCTA css-1wgos6z-watchLiveCTA"><span class="css-1hxk0l6-watchLiveCTAText"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 32 32" width="12" height="12" focusable="false" aria-hidden="true"><path d="M29 16 5.8 1v30z"></path></svg>Watch</span></div>',
+    },
+    {
+      title: 'Displays a title as is when there is punctuation.',
+      inputTitle: 'Title with punctuation!',
+      expectedResult:
+        '<div><span class="hoverStylesText css-1hja70-mediaDescription-openMediaDescription-description"><span class="css-1cxhpmz-Text">Title with punctuation!</span><span class="css-3g34uu-Text"> CBBC</span></span><span class="css-10lim9y-guidanceMessage" data-testid="warning-message"><span class="css-49uz87-visuallyHiddenText">, </span>Contains some upsetting scenes.</span></div><div class="hoverStylesCTA css-1wgos6z-watchLiveCTA"><span class="css-1hxk0l6-watchLiveCTAText"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 32 32" width="12" height="12" focusable="false" aria-hidden="true"><path d="M29 16 5.8 1v30z"></path></svg>Watch</span></div>',
+    },
+  ])('Open state - $title', ({ inputTitle, expectedResult }) => {
+    const mediaBlock = fixtureData[0];
+    mediaBlock.model.synopses.short = inputTitle;
+
+    const { container } = render(
+      <LiveHeaderMedia mediaCollection={fixtureData as MediaCollection[]} />,
+    );
+
+    const mediaLoader = container.querySelector(
+      'button[data-testid="watch-now-close-button"]',
+    );
+
+    expect(mediaLoader?.innerHTML).toEqual(expectedResult);
```

Comment:
Would it be better to use innerText instead of innerHTML, so that the test is more readable?

Reply:
It doesn't support innerText

Reply:
We can use textContent instead!

```suggestion
    expect(mediaLoader?.textContent).toEqual(expectedResult);
```